code/modules/modular_computers/networking/_network.dm

```diff
@@ -24,6 +24,8 @@
 	var/datum/money_account/parent/network/parent_account
 	var/list/datum/extension/network_device/money_cube/money_cubes = list()
 
+	var/datum/extension/network_device/trade_controller/trade_controller
```

Comment:
Just a note, but if you include this in the base computer network, you'll have to include the trade beacon stuff into nebula.dme so the unit tests don't fail.

Reply:
I think you actually just did..